src/test/java/com/google/javascript/clutz/DeclarationGeneratorTests.java

```diff
@@ -60,6 +69,15 @@ static File getGoldenFile(final File input) {
     return new File(input.getPath().replaceAll("\\.js$", ".d.ts"));
   }
 
+  static String maybeExternFileName(String testFileName) {
+    String possibleFileName = testFileName.replace(".js",".externs.js");
+    Path testDir = FileSystems.getDefault().getPath("src", "test", "java");
+    String packageName = DeclarationGeneratorTests.class.getPackage().getName();
```

Comment:
I feel like I've seen this package code below, extract a method `getFileInPackage`?